README.md

```diff
@@ -2,6 +2,3 @@
 
 This is the repository for Storage Buckets. You're welcome to
 [contribute](CONTRIBUTING.md)!
-
-* [The explainer](https://github.com/WICG/storage-buckets/blob/main/explainer.md) is a developer-oriented preview of the API.
-* [The specification draft](https://wicg.github.io/storage-buckets/) is aimed at browser developers.
```

Comment:
Did you mean to make this change?

Reply:
I'm not sure how this change or the one to index.html appeared but I've reverted them.